core/cmd/client.go

```diff
@@ -217,6 +218,12 @@ func (n ChainlinkAppFactory) NewApplication(ctx context.Context, cfg config.Gene
 		}
 	}
 
+	// Configure and optionally start the audit log forwarder service
+	auditLogger, err := audit.NewAuditLogger(appLggr, cfg.AuditLoggerConfig())
+	if err != nil {
+		appLggr.Info("Audit logger could not be started. err: ", err)
```

Comment:
```suggestion
		appLggr.Infow("Audit logger could not be started", "err", err)
```

Can you check other uses of logger to make sure they output structured logging correctly?

Reply:
Actually, I would recommend failing application boot if this thing fails to start.

Reply:
It also needs to be closed on application exit.

Reply:
Since this is a subservice will the that be handled automatically during shutdown where close is called? The requests now use context.Context so should be killed when shutdown is requested.

Reply:
Yes it will be started/closed as part of the slice variable named `subservices`, but it is misnamed: https://github.com/smartcontractkit/chainlink/pull/7487